ts/IterableInboxCustomizations.ts

```diff
@@ -36,11 +36,11 @@ type IterableInboxCustomizations = {
       paddingBottom: number
    },
  
-   body: {
-      fontSize: number,
-      color: string,
-      paddingBottom: number
-   },
+   // body: {
```

Comment:
Delete?

Reply:
This will be deleted and addressed on PR-220.